### DISCUSSION TITLE
NextJS 13 ready for production

### DISCUSSION THREAD

**Discussion by aleczratiu** [vercel/next.js#57244] [Help]
### Summary

Hi, a lot of debates and discussions about this, is nextjs 13 ready for production? 🙂

**Comment by aleczratiu** [comment]
up?

**Comment by icyJoseph** [comment]
Hi, I would say, for the most part yes. 

I think the community has narrowed down a few quirks and issues and such into this thread: https://www.reddit.com/r/nextjs/comments/13kwcax/the_app_router_is_not_productionready_yet/ where Lee Robinson has tried to address some of the outlined quirks and issues. **Read through carefully**!

Now tomorrow the Next.js Conf, might bring some news and features that could ease some of those problems.

**Reply by aleczratiu** [reply]
I see, thank you @icyJoseph, see you at Next.js Conf ✌️

**Comment by landsman** [comment]
It is. Just do not use too many 3rd-party dependencies which are not :) minimalism FTW.

**Reply by aleczratiu** [reply]
Yup, already using only those that support 13. Thanks @landsman 👍
